common/test/acceptance/pages/studio/course_info.py

```diff
@@ -1,16 +1,144 @@
 """
 Course Updates page.
 """
-
+from common.test.acceptance.pages.common.utils import click_css, confirm_prompt
 from common.test.acceptance.pages.studio.course_page import CoursePage
+from common.test.acceptance.pages.studio.utils import type_in_codemirror, set_input_value
 
 
 class CourseUpdatesPage(CoursePage):
     """
     Course Updates page.
     """
-
     url_path = "course_info"
 
     def is_browser_on_page(self):
-        return self.q(css='body.view-updates').present
+        """
+        Returns whether or not the browser on the page and has loaded the required content
+        """
+        # Check for the presence of handouts-content, when it is present the render function has completed
+        # loading the updates and handout sections
+        return self.q(css='.handouts-content').present
+
+    def are_course_updates_on_page(self):
+        """
+        Is the course updates content Div loaded on the page?
+        """
+        return self.q(css='article#course-update-view.course-updates').present
+
+    def course_update_list_empty(self):
```

Comment:
Nit: you've been using a Java-style of starting boolean methods with "is". I can buy into that, but if you want to use that style, be consistent. This method would then be is_course_update_list_empty.